modules/andes-core/broker/src/main/java/org/wso2/andes/subscription/AMQPLocalSubscription.java

```diff
@@ -88,6 +98,8 @@ public AMQPLocalSubscription(AMQQueue amqQueue, Subscription amqpSubscription, b
 
         this.isDurable = isDurable;
         this.isBoundToTopic = isBoundToTopic;
+
+        this.storedMessageCache = new HashMap<>();
```

Comment:
Currently no. Because invoke path will be SlotDeliveryWorker > Message Flusher > Delivery strategy > Check Selector rule.